app/imdb_tools.py

```diff
@@ -161,6 +162,44 @@ def tmdb_get_movies(config_path, plex, data, is_list=False):
 
     return matched, missing
 
+def get_tautulli(config_path, plex, data):
+    tautulli = config_tools.Tautulli(config_path)
+    type = config_tools.check_for_attribute(data, "list_type", parent="tautulli", test_list=["popular", "watched"], options="| \tpopular (Most Popular List)\n| \twatched (Most Watched List)", throw=True, save=False)
```

Comment:
change `type` to something else, since it's a Python built-in

Reply:
fixed